lib/pi_piper/pin.rb

```diff
@@ -174,6 +179,18 @@ def released?
 
     private
 
+    def valid_trigger?
+      [:rising, :falling, :both].include?(@trigger)
```

Comment:
True, I think we can move the validations to the driver. I just moved the code around here and added the check for a valid pin value. I will add this to the driver itself next.